protocol/lib/metrics/constants.go

```diff
@@ -100,6 +100,11 @@ const (
 	CancelShortTermOrder                         = "cancel_short_term_order"
 	CancelStatefulOrder                          = "cancel_stateful_order"
 	ClobPairId                                   = "clob_pair_id"
+	ClobPrepareCheckState                        = "clob_prepare_check_state"
+	ClobLiquidateSubaccountsAgainstOrderbook     = "clob_liquidate_subaccounts_against_orderbook"
+	ClobLiquidateSubaccounts_GetLiquidations     = "clob_liquidate_subaccounts_against_orderbook_get_liquidations"
+	ClobLiquidateSubaccounts_PlaceLiquidations   = "clob_liquidate_subaccounts_against_orderbook_place_liquidations"
+	ClobLiquidateSubaccounts_Deleverage          = "clob_liquidate_subaccounts_against_orderbook_deleverage"
```

Comment:
I think we can drop the clob prefix. it's kind of verbose now

Reply:
If it's fine I'm going to leave `ClobPrepareCheckState` +  `ClobLiquidateSubaccountsAgainstOrderbook ` as variable names since removing the `Clob` prefix causes them to conflict with function names.

I'll remove it from the other ones though + remove it from all stat names since agreed it's redundant